src/redux/wallets/selectors.ts

```diff
@@ -78,6 +78,20 @@ export const getAccounts = (state: IReduxState, blockchain: Blockchain): IAccoun
     return accounts;
 };
 
+export const selectedBlockchainHasAccounts = (state: IReduxState): boolean => {
```

Comment:
i think it would be more useful to have a selector: `getSelectedBlockchainAccounts(state): IAccountState[]` that will return the accounts of the selected blockchain. Your check will become:
`getSelectedBlockchainAccounts().length > 0`